.gitignore

```diff
@@ -40,8 +40,8 @@ nosetests.xml
 .idea/
 
 # subject files
-examples/funloc/subj*/**/
-examples/funloc/subj_*/**/*.html
+/examples/funloc/subj01/
+/examples/funloc/subj02/
```

Comment:
these changes look incorrect -- why add the leading slash, and why change to `subj01` when the name is `subj_01`?

Reply:
Sorry that's a typo from my end. The leading slash was inserted by pycharm.